packages/vite/src/node/plugins/css.ts

```diff
@@ -1640,7 +1641,8 @@ async function resolvePostcssConfig(
   } else {
     const searchPath =
       typeof inlineOptions === 'string' ? inlineOptions : config.root
-    result = postcssrc({}, searchPath).catch((e) => {
+    const stopDir = searchForWorkspaceRoot(config.root)
+    result = postcssrc({}, searchPath, { stopDir }).catch((e) => {
```

Comment:
📝 Note for reviewers: I tested that this works even if `searchPath` is outside `stopDir`. (It seems `stopDir` gets ignored in that case).

Reply:
Yeah I'm guessing that `stopDir` is only took into account when the searching crawls up one directory above.